cmd/tfchainc/erc20cmd.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"reflect"
+
+	"github.com/spf13/cobra"
+	"github.com/threefoldfoundation/tfchain/cmd/tfchainc/internal"
+	"github.com/threefoldtech/rivine/pkg/cli"
+)
+
+// createERC20Cmd creates rootcommand for ERC20 and adds a subcommand
+// if rootcommand executed the user will also see the output of the syncing status of ethereum
+func createERC20Cmd(client *internal.CommandLineClient) *cobra.Command {
+	erc20SubCmds := &erc20SubCmds{cli: client}
+
+	// define Rootcommand
+	var (
+		rootCmd = &cobra.Command{
+			Use:   "erc20",
+			Short: "Perform erc20 actions",
+			Long:  "Perform erc20 actions",
+			Run:   erc20SubCmds.getSyncingStatus,
+		}
+		getSyncingStatusCmd = &cobra.Command{
+			Use:   "syncstatus",
+			Short: "Get the ethereum sync status",
+			Long:  `Get the ethereum chain sync status.`,
+			Run:   erc20SubCmds.getSyncingStatus,
+		}
+	)
+
+	rootCmd.AddCommand(getSyncingStatusCmd)
+
+	// register flags
+	getSyncingStatusCmd.Flags().Var(
+		cli.NewEncodingTypeFlag(0, &erc20SubCmds.getSyncingStatusCfg.EncodingType, 0), "encoding",
```

Comment:
`0` allows every kind of encoding type, we only want to allow JSON and Human-readable though.

Reply:
Changed in: https://github.com/threefoldfoundation/tfchain/pull/307/commits/5e3a7be7c727594b63955708990632c0e1ddce8f